### DISCUSSION TITLE
Update Crashlytics SDK

### DISCUSSION THREAD

**Discussion by vernalclareai** [firebase/flutterfire#3371] [Feature request] [answered]
firebase_crashlytics 0.1.4+1
Getting warning in firebase console.
![image](https://user-images.githubusercontent.com/61595318/91376628-1674c300-e850-11ea-9f06-ac464835a735.png)

Do you have a plan to update the firebase_crashlytics?

Thanks.

**Comment by Salakar** [comment]
Yes, and it's almost done. Look out for it next week. Thanks

**Reply by vernalclareai** [reply]
Sounds great.

**Comment by Salakar** [comment, marked as answer]
This has now landed on master :)

Checkout the migration guide to get started on upgrading to the new Crashlytics plugin: https://firebase.flutter.dev/docs/migration/
